style.css

```diff
@@ -0,0 +1,450 @@
+.header-background {
+  position: absolute;
+  width: 475px;
+  height: 916px;
+  left: 0;
+  top: 1px;
+
+  /* Neutrals/N0 */
+
+  background-color: #fff;
+}
+
+.dkl {
+  float: right;
+  margin-right: 90px;
+}
+
+.sec3rlink23 {
+  list-style-type: none;
+}
+
+.dkk {
+  position: absolute;
+  width: 48px;
+  height: 48px;
+  left: 12px;
+}
+
+.dd {
+  display: inline;
+}
+
+.limage {
+  display: inline;
+  list-style-type: none;
+}
+
+.dk {
+  margin-top: 20px;
+  margin-left: 30px;
+}
+
+.sa {
+  margin-left: 30px;
+}
+
+.cars {
+  margin-left: 160px;
+  border: 0;
+}
+
+.m-logo {
+  padding-top: 10px;
+  padding-left: 10px;
+  text-decoration: none;
+  width: 76px;
+  height: 20px;
+  font-family: Arial, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+
+  /* identical to box height, or 111% */
+
+  display: flex;
+  align-items: center;
+  text-align: center;
+
+  /* Primary/B400 */
+
+  color: #6070ff;
+
+  /* Inside auto layout */
+
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+a.inline {
+  width: bold;
+  padding-left: 20px;
+  text-decoration: none;
+  list-style-type: none;
+  padding-right: 0;
+  float: right;
+  color: #6070ff;
+}
+
+.link-right {
+  margin-top: -30px;
+  text-decoration: none;
+  list-style-type: none;
+  float: right;
+}
+
+.section1 {
+  width: 375px;
+  padding-top: 55px;
+  margin-top: 30px;
+  display: inline;
+  padding-right: 60px;
+}
+
+.l1 {
+  margin-top: 10px;
+}
+
+.section2 {
+  padding-left: 10px;
+  width: 920px;
+  padding-top: 50px;
+  height: 122px;
```

Comment:
- [ ] Kindly make sure you design is spaced and looks as close as possible to the Figma design 🙏 

Figma Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/184996674-69b30a02-6aea-40b7-b54c-315dbb920b8d.png)  |  ![image](https://user-images.githubusercontent.com/74814780/184996707-b2668f95-9546-4893-9052-7e32402aa0b1.png)